recipes/mpir/all/conanfile.py

```diff
@@ -88,15 +98,15 @@ def _vcxproj_paths(self):
         return vcxproj_paths
 
     def _build_visual_studio(self):
-        if "MD" in self.settings.compiler.runtime and not self.options.shared: # RuntimeLibrary only defined in lib props files
-                props_path = os.path.join(self._source_subfolder, "build.vc",
-                "mpir_{}_{}.props".format(str(self.settings.build_type).lower(), self._dll_or_lib))
-                if self.settings.build_type == "Debug":
-                    tools.replace_in_file(props_path, "<RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>",
-                                                      "<RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>")
-                else:
-                    tools.replace_in_file(props_path, "<RuntimeLibrary>MultiThreaded</RuntimeLibrary>",
-                                                      "<RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>")
+        if "MD" in msvc_runtime_flag(self) and not self.options.shared: # RuntimeLibrary only defined in lib props files
+            props_path = os.path.join(self._source_subfolder, "build.vc",
+            "mpir_{}_{}.props".format(str(self.settings.build_type).lower(), self._dll_or_lib))
+            if self.settings.build_type == "Debug":
+                tools.replace_in_file(props_path, "<RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>",
+                                                  "<RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>")
+            else:
+                tools.replace_in_file(props_path, "<RuntimeLibrary>MultiThreaded</RuntimeLibrary>",
+                                                  "<RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>")
```

Comment:
I think the runtime does not necessarily depend on the build_type.
```suggestion
            if "d" in msvc_runtime_flag(self):
                tools.replace_in_file(props_path, "<RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>",
                                                  "<RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>")
            else:
                tools.replace_in_file(props_path, "<RuntimeLibrary>MultiThreaded</RuntimeLibrary>",
                                                  "<RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>")
```